sdk/include/psdk/gdiplusheaders.h

```diff
@@ -877,7 +877,8 @@ class Region : public GdiplusBase
     Status
     Complement(const GraphicsPath *path)
     {
-        return SetStatus(DllExports::GdipCombineRegionPath(region, path ? path->path : NULL, CombineModeComplement));
+        return SetStatus(
+            DllExports::GdipCombineRegionPath(region, path ? path->nativePath : NULL, CombineModeComplement));
```

Comment:
newline?

Reply:
`clang-format` does format like this...

Reply:
I don't know what's the current max line size. How do you think, which number should I put?

Reply:
OK.

Reply:
https://reactos.org/wiki/Coding_Style
No max line size specified. Maybe 120 characters?

Reply:
https://wiki.winehq.org/Submitting_Patches
>Avoid very long lines (there is no hard limit, the preferred length is 100, but 120 or 80 chars are fine too)